src/components/common/Directors.vue

```diff
@@ -814,7 +814,7 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
           this.original.forEach((director, i) => {
             director.id = i + 1
             director.isFeeApplied = (director.isFeeApplied !== undefined) ? director.isFeeApplied : false
-            director.isDirectorActionable = (director.cessationDate === null)
+            director.isDirectorActionable = !director.cessationDate
```

Comment:
This was a bug identified in the unit tests: `cessationDate` in this case was undefined (not null) so the logic didn't work.

Ditto on line 994 below.